src/Flicking.ts

```diff
@@ -53,6 +53,7 @@ export interface FlickingOptions {
   adaptive: boolean;
   // EVENT
   needPanelThreshold: number;
+  noEventsBeforeInit: boolean;
```

Comment:
Longer name than `noEventsBeforeInit`, but what about `preventEventsBeforeInit`?

Reply:
I chose that first time, but I've changed like that as you guys think it'd too long and thought it could be confusing with native `preventDefault`
@daybrush How do you think?

Reply:
If the name itself is not the same, there seems to be no problem. `preventEventsBeforeInit` is good.

Reply:
Well okay then, I'll change it like that 👍